src/example/followline.ts

```diff
@@ -0,0 +1,124 @@
+import { getSensorData, init, move, sleep } from '../lib';
+
+require('dotenv').config();
```

Comment:
```suggestion
import {config as dotenv} from "dotenv"

dotenv()
```

typescript doesn't like require() trust me I tried